docs/user-guide/downward-api/index.md

```diff
@@ -54,12 +74,52 @@ downward API:
 
 {% include code.html language="yaml" file="dapi-pod.yaml" ghlink="/docs/user-guide/downward-api/dapi-pod.yaml" %}
 
+This is another example of a pod that consumes its container's resources
+via the downward API:
+
+```yaml
```

Comment:
This seems like a clear case where it would be better to put this YAML in a separate file and use the widget to include it (so it can be directly downloaded/copied to clipboard). See line 75 of this file.